react-ui/src/pages/entry/EntryModal.jsx

```diff
@@ -122,8 +116,15 @@ function EntryModal() {
   };
 
   /**
-   * Navigation
+  * Layout
+  */
+  const classes = useStyles(entry);
+  const imageContainerRef = useRef(null);
+
+  /**
+   * Navigation and voting
    */
+  const { changeRating, clearRating } = useVoting(entryId);
```

Comment:
‼️ MAJOR BUG ‼️ 

`useVoting` is only instantiated once when the component is rendered, and these functions always apply to the `entryId` of the first entry loaded.

Reply:
Good catch, I hadn't caught that `handleKeyUp` was in a useEffect. I added it to the dependencies array.